syntax = "proto3";

package SailGame;

service Hello {
  rpc BiStream(stream IntWrapper) returns (stream IntWrapper) {}
}

message IntWrapper {
  int32 payload = 1;
}
 